### ISSUE TITLE
Image-builder scw version is out of date.

### ISSUE THREAD

**Issue by ewoutp** [scaleway/image-builder#7]
During a `make image_on_local` I got this

```
[+] Creating new server in rescue mode with a secondary volume...
FATA[0000] cannot execute 'run': failed to create server: json: cannot unmarshal string into Go value of type api.ScalewayKernel
```

I'm updating `scw` manually which should fix it.

**Comment by ewoutp** [comment]
Suggestion, update `scw` in the `image-builder-config` step.

**Comment by QuentinPerez** [comment]
Hi @ewoutp, thanks for your feedback 😊

We have released the version 1.4.1 of image builder with the latest version of scw

Or if you don't want to launch a new instance, you can upgrade scw manually [releases](https://github.com/scaleway/scaleway-cli/releases/tag/v1.8.1)

**Comment by moul** [comment]
This can be closed, @ewoutp please reopen if needed
